identity-service/src/txRelay.js

```diff
@@ -24,37 +24,32 @@ async function delay (ms) {
   return new Promise(resolve => setTimeout(resolve, ms))
 }
 
+async function getGasPrice (req, web3) {
+  let gasPrice = parseInt(await web3.eth.getGasPrice())
+  if (isNaN(gasPrice) || gasPrice > HIGH_GAS_PRICE) {
+    req.logger.info('txRelay - gas price was not defined or was greater than HIGH_GAS_PRICE', gasPrice)
```

Comment:
i think having the logger also log the value of HIGH_GAS_PRICE would be helpful too